docs/upgrade.html

```diff
@@ -38,6 +38,12 @@ <h5><a id="upgrade_400_notable" href="#upgrade_400_notable">Notable changes in 4
             taken when it comes to kafka clients that are not part of Apache Kafka, please see
             <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-896%3A+Remove+old+client+protocol+API+versions+in+Kafka+4.0">KIP-896</a> for the details.
         </li>
+        <li>
+            Before updating your Kafka cluster which build on Zookeeper to Kafka 4.0, you must understand what the updates and changes 
+            are between Zookeeper mode and Kraft mode. See the following page to learn about some of the significant changes in 
+            latest Kafka releases.
+            <!--#include virtual="zk2kraft.html" -->
```

Comment:
Hmm what does `#include` do? Does it embed the page? If so, that doesn't seem like the right thing. We'd want to link to a separate page, right?

Reply:
Thanks!